frontend/src/layout/panel-layout/ProjectTree/projectTreeLogic.tsx

```diff
@@ -112,6 +114,32 @@ export const projectTreeLogic = kea<projectTreeLogicType>([
                 },
             },
         ],
+        archivedFolder: [
+            false as boolean,
+            {
+                loadArchivedFolder: async () => {
+                    try {
+                        const response = await api.fileSystem.list({ path: 'Archived' })
+                        if (response.results.length === 0) {
+                            await api.fileSystem.create({
+                                id: PROJECT_TREE_ARCHIVE_NAME,
+                                path: PROJECT_TREE_ARCHIVE_NAME,
+                                type: 'folder',
+                            })
+                        }
```

Comment:
I think you can get rid of this part now that there's a fake folder in the reducer and save on an API call.